repos/fdbt-site/src/pages/fareConfirmation.tsx

```diff
@@ -167,14 +169,14 @@ export const buildFareConfirmationElements = (
                 confirmationElements.push({
                     name: `Time restrictions - ${sentenceCaseString(fullTimeRestriction.day)}`,
                     content: `Start time: ${timeBand.startTime || 'N/A'} End time: ${timeBand.endTime || 'N/A'}`,
-                    href: 'defineTimeRestrictions',
+                    href: `${globalSettingsEnabled ? 'selectTimeRestrictions' : 'defineTimeRestrictions'}`,
```

Comment:
FYI you don't need the template strings here or in the one below

Reply:
fixed